packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -1,13 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import React, { useState, ReactNode, FormEvent, useCallback, useRef } from 'react';
-/* @conditional-compile-remove(mention) */
-import { useEffect, useMemo } from 'react';
-/* @conditional-compile-remove(mention) */
-import { ComponentStrings, useLocale } from '../localization';
-/* @conditional-compile-remove(mention) */
-import { Announcer } from './Announcer';
+import React, { useState, ReactNode, FormEvent, useCallback } from 'react';
```

Comment:
Is `useState` still used in this file after refactoring?

Reply:
Yes - for the `isHover`